tree/ntuple/doc/BinaryFormatSpecification.md

```diff
@@ -799,6 +800,58 @@ In every cluster, every field has exactly one primary column representation.
 All other representations must be suppressed.
 Note that the primary column representation can change from cluster to cluster.
 
+## Linked Attribute Sets
+
+An RNTuple may have zero or more linked Attribute Sets, containing metadata.
```

Comment:
```suggestion
An RNTuple may have zero or more linked Attribute Sets, containing user-defined metadata.
```

Reply:
In the end I think "user-defined" is fine. @jblomer should the specs explicitly reserve the "ROOT" attribute namespace?